CSS/style.css

```diff
@@ -24,24 +24,47 @@
   font-weight: 600;
   font-size: 1.5rem;
 }
-.component-container {
+
+.content {
   margin-top: 20px;
-  /* border: 1px solid red; */
+  border: 1px solid blue;
   display: flex;
+  width: 100%;
 }
 
-.component-content {
-  max-width: 100%;
-  flex: 0 0 700px;
-  margin: 0 auto;
-  /* border: 1px solid blue; */
-  margin-left: 350px;
+.side-nav {
+  flex: 0 0 250px;
+  margin: 1rem;
+}
+.component-container {
+  flex: 1 1 0%;
```

Comment:
flex: 1; is same as flex: 1 1 0%;
don't need to change it here, but its nice to remember

Reply:
Okay, thanks for that